egs/csj/s5/local/chain/tuning/run_tdnn_7k.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
```

Comment:
please rename to run_tdnn_1a.sh and have the experimental directory be something like tdnn_1a_sp (assuming speed_perturb is true).